signature-crate/src/lib.rs

```diff
@@ -1 +1,22 @@
+//! Digital signature types and traits
+
 #![crate_name = "signature"]
+#![crate_type = "lib"]
```

Comment:
Is there a reason for `crate_name` and `crate_type` attributes? I see it sometimes, but I don't know about why are they used.

You also can add `#![doc(html_logo_url ="https://raw.githubusercontent.com/RustCrypto/meta/master/logo_small.png")]` for the nice logo in docs. :)

Reply:
Perhaps this is pointless boilerplate. Also yes I know about `html_logo_url` 😉